### PR TITLE
HUGE PR: Code refactoring + prettier + copy button + bug fix + redesign +

### PR DESCRIPTION
**Pull request by Looskie**
redesign these

### CODE REVIEW

**Comment by cnrad** [comment]
do better :/ ¯\_(ツ)_/¯

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "quoteProps": "consistent",
+  "printWidth": 120,
+  "bracketSpacing": true,
+  "singleQuote": false,
+  "useTabs": false,
+  "arrowParens": "avoid",
+  "tabWidth": 4
```
Comment (Phineas) [inline .prettierrc:8]: personal op that this should be 2

Comment (Looskie) [reply .prettierrc:8]: @Phineas i was doing it to his personal preference, i love 2

Comment (cnrad) [reply .prettierrc:8]: why do you goblins use 2

Comment (barbarbar338) [reply .prettierrc:8]: 4 rocks

**Comment by cnrad** [comment]
thank you for the fire redesign

**Comment by Looskie** [comment]
nppp :))))

**Pull request merged**